tomviz/ExternalPythonExecutor.h

```diff
@@ -0,0 +1,48 @@
+/* This source file is part of the Tomviz project, https://tomviz.org/.
+   It is released under the 3-Clause BSD License, see "LICENSE". */
+
+#ifndef tomvizExternalPythonExecutor_h
+#define tomvizExternalPythonExecutor_h
+
+#include "PipelineExecutor.h"
+
+#include <QObject>
+#include <QProcess>
+#include <QScopedPointer>
+
+namespace tomviz {
+
+class DataSource;
+class Operator;
+class Pipeline;
+
```

Comment:
Same here, perhaps "Execute the pipeline in a specified external Python environment in order to enable GPU acceleration, custom packages, etc."

Reply:
Done